tests/integration/helpers.py

```diff
@@ -744,7 +744,7 @@ async def switchover(
         candidate: The unit that should be elected the new primary.
     """
     primary_ip = await get_unit_address(ops_test, current_primary)
-    for attempt in Retrying(stop=stop_after_attempt(4), wait=wait_fixed(5), reraise=True):
+    for attempt in Retrying(stop=stop_after_attempt(60), wait=wait_fixed(3), reraise=True):
```

Comment:
probably a bit overkill, but this was affecting `test_backups` which is an annoying test to retry and these values helped.

Reply:
In this PR commits there are 3 examples:
https://github.com/canonical/postgresql-k8s-operator/actions/runs/10894200338/job/30260414050#step:31:395